### PR TITLE
Feat/docs install

### PR DESCRIPTION
**Pull request by nnixaa**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

### CODE REVIEW

File: src/framework/theme/components/badge/badge.component.ts [hunk 1]
```diff
@@ -16,6 +16,19 @@ import { NbLayoutDirectionService } from '../../services/direction.service';
  * It means parent `position` should be set to anything except `static`, e.g. `relative`,
  * `absolute`, `fixed`, or `sticky`.
  *
+ * ### Installation
+ *
+ * Import `NbCalendarModule` to your feature module.
```
Comment (Tibing) [inline src/framework/theme/components/badge/badge.component.ts]: I don't think you have to import `NbCalendarModule` if you want to use badges)

File: src/framework/theme/components/calendar/calendar-range.component.ts [hunk 2]
```diff
@@ -22,6 +22,18 @@ export interface NbCalendarRange<D> {
  * ```html
  * <nb-calendar-range [(range)]="range"></nb-calendar-range>
  * ```
+ * ### Installation
+ *
+ * Import `NbCalendarModule` to your feature module.
```
Comment (Tibing) [inline src/framework/theme/components/calendar/calendar-range.component.ts]: Not `NbCalendarModule` but `NbCalendarRangeModule`

File: src/framework/theme/components/calendar/calendar.component.ts [hunk 3]
```diff
@@ -20,6 +20,20 @@ import { NbCalendarCell, NbCalendarSize, NbCalendarViewMode } from '../calendar-
  * Basic usage example
  * @stacked-example(Showcase, calendar/calendar-showcase.component)
  *
+ * ### Installation
+ *
+ * Import `NbCalendarRangeModule` to your feature module.
```
Comment (Tibing) [inline src/framework/theme/components/calendar/calendar.component.ts]: And here not `NbCalendarRangeModule` but `NbCalendarModule`

Comment (Tibing) [reply src/framework/theme/components/calendar/calendar.component.ts]: Maybe we can merge calendar and calendar range modules? Do we really need this separation?

File: package.json [hunk 4]
```diff
@@ -109,10 +109,10 @@
     "@angular-devkit/build-angular": "~0.6.0",
     "@angular-devkit/core": "^0.6.5",
     "@angular-devkit/schematics": "^0.6.5",
+    "@angular/cdk": "6.0.0",
```
Comment (Tibing) [inline package.json]: Is it important change?

File: src/framework/theme/components/calendar/calendar.component.ts (hunk 3 above)
Comment (nnixaa) [reply src/framework/theme/components/calendar/calendar.component.ts]: let's keep both for now

File: package.json (hunk 4 above)
Comment (nnixaa) [reply package.json]: it is automatic by npm, so it will appear now or in future

**Pull request merged**